src/Symfony/Component/DependencyInjection/Container.php

```diff
@@ -113,6 +121,21 @@ public function getParameterBag()
         return $this->parameterBag;
     }
 
+    public function setEnvironmentBag(EnvironmentBagInterface $environmentBag)
+    {
+        $this->environmentBag = $environmentBag;
+    }
+
+    public function getEnvironmentBag()
+    {
+        return $this->environmentBag;
+    }
```

Comment:
What's the use case for this getter and setter? They're not used by any code added in this PR.